test/blocks/table/table.test.js

```diff
@@ -111,5 +114,15 @@ describe('table and tablemetadata', () => {
       expect(tooltipHeading.childNodes.length).to.equal(2);
       expect(tooltipHeading.querySelector('.milo-tooltip, .icon-tooltip')).to.exist;
     });
+
+    it('should apply aria-label to all selects within .filters on mobile', async () => {
+      window.innerWidth = 375;
+      window.dispatchEvent(new Event('resize'));
+      await delay(500);
+      const selectElements = document.querySelectorAll('.filters select');
+      selectElements.forEach((selectElement) => {
+        expect(selectElement.getAttribute('aria-label')).to.equal('choose table column');
```

Comment:
Again, we should probably mock the placeholders logic/values, so we might need to adapt the expected value here as well.